ui/raidboss/data/05-shb/trial/ruby-weapon-ex.js

```diff
@@ -343,11 +347,13 @@
         if (matches.y < 90) {
           return {
             en: 'Comets N/S',
+            fr: 'Comètes N/S',
             ko: '남/북 운석 낙하',
           };
         } else if (matches.x < 90) {
           return {
             en: 'Comets E/W',
+            fr: 'Comètes E/O',00
```

Comment:
It looks like this `00` here is the problem, based on the error messages from Travis.

Reply:
oo, ty my son have touch at the moment my macro. sorry really.

Reply:
No apologies needed! :)